plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlValidator.java

```diff
@@ -89,7 +101,27 @@ void validate(IFile resource, byte[] bytes)
         }
       }
     } catch (SAXException ex) {
-      createSaxErrorMessage(resource, ex);
+      createSaxErrorMessage(resource, (SAXParseException) ex.getException());
```

Comment:
No, but that's not likely to occur. SAXParseException is a subclass of SAXParseException.